### PR TITLE
Update _introduction.md

### PR DESCRIPTION
**Pull request by KseniiaNovinskaia**
I have updated the into part with all the key concepts. Can you quickly review?

### CODE REVIEW

File: source/includes/_introduction.md [hunk 1]
```diff
@@ -12,159 +12,359 @@ You can use our API to access Distribusion content and book bus rides. We have p
...
-![alt text](/images/API_V4.0_workflow.png)
 
-## Integration Workflow
+## Stations, Cities, and Areas
 
-1. _(1-4 weeks)_ **Integrate our API**, using this documentation. 
-If you have any questions, do not hesitate to write us at [partner@distribusion.com](mailto:partner@distribusion.com)
+The [#stations](https://docs.distribusion.com/#stations7) endpoint returns all the stations available in the Distribusion API, as well as detailed information for each, including: 
+ - type
+- name
```
Comment (lesin) [inline source/includes/_introduction.md:26]:
```suggestion
- type
- name
```

**Review by lesin — APPROVED** [review]

**Comment by lesin** [comment]
@KseniiaNovinskaia do you know how to deploy it? I see the image build is failing, but not sure if you need it for this app to be deployed

**Comment by KseniiaNovinskaia** [comment]
> @KseniiaNovinskaia do you know how to deploy it? I see the image build is failing, but not sure if you need it for this app to be deployed

I am not sure I need to do anything with the image.

**Comment by joerghaubrichs** [comment]
@KseniiaNovinskaia I just saw the failing build, so had a quick look at the changes, and saw that we apparently now support Bitcoin and pure Gold as currencies? :D Maybe something we should change before going live with it. That list seems not to be filtered for active or so..

**Comment by KseniiaNovinskaia** [comment]
Good point Jorg, I just used the link once shared on slack
https://currencylayer.com/currencies but didn't go through the whole list.

Will change it.

Kseniia

**Pull request merged**